lib/generators/pageflow/routes/routes_generator.rb

```diff
@@ -7,9 +7,19 @@ class RoutesGenerator < Rails::Generators::Base
 
       def add_route
         inject_into_file 'config/routes.rb', after: "  ActiveAdmin.routes(self)\n" do
-          "  Pageflow.routes(self)\n"
+          <<-HEREDOC
+  Pageflow.routes(self)
+
+  authenticate :user, lambda {|user| user.admin?} do
+    mount Resque::Server.new, at: "/background_jobs"
+  end
+          HEREDOC
         end
       end
+
+      def require_resque_server
+        prepend_to_file 'config/routes.rb', "require 'resque/server'\n\n"
```

Comment:
Let's also require `resque_scheduler/server`. It adds another tab to the Resque web interface.